src/server/pps/server/api_server.go

```diff
@@ -823,6 +823,13 @@ func (a *apiServer) GetLogs(request *pps.GetLogsRequest, apiGetLogsServer pps.AP
 		if err != nil {
 			return fmt.Errorf("could not get job information for %s: %s", request.Job.ID, err.Error())
 		}
+
+		// If the job had stats enabled, we use the logs from the stats
+		// commit since that's likely to yield better results.
+		if jobInfo.EnableStats && jobInfo.StatsCommit != nil {
```

Comment:
You could just check `StatsCommit != nil` here. It being non-nil implies that `EnableStats` must have been true too.